frontend/src/lib/i18n/locales/en.json

```diff
@@ -263,6 +263,10 @@
     "shared": {
       "greeting": "Hello {name}!",
       "or_nvaigate_to_url": "Or navigate to this URL in your browser:"
+    },
+    "create_project_request_email": {
```

Comment:
I'm not quite sure how to word it, but it should be more obvious that the user is actually requesting an admin to create the project for them. They're not requesting "to create a project", they're requesting to have a project created on their behalf.
Maybe:
Subject: Project request {project-name}
Heading: "User {name} ({email}) has requested that the following project be created for them:"